src/audio/google/Kconfig

```diff
@@ -32,21 +33,15 @@ config COMP_GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ
 	  Sets the sample rate for the memory buffer for the Google real-time
 	  communication audio processing.
 
-config COMP_GOOGLE_RTC_AUDIO_PROCESSING_NUM_CHANNELS
-	depends on COMP_GOOGLE_RTC_AUDIO_PROCESSING
-	int "Number of channels to process for Google Real Time Communication Audio processing"
-	default 1
-	help
-	  Sets the number of channels to process in the Google real-time
-	  communication audio processing.
-
-config COMP_GOOGLE_RTC_AUDIO_PROCESSING_NUM_AEC_REFERENCE_CHANNELS
-	depends on COMP_GOOGLE_RTC_AUDIO_PROCESSING
-	int "Number of AEC reference channels for Google Real Time Communication Audio processing"
+config COMP_GOOGLE_RTC_AUDIO_PROCESSING_CHANNEL_MAX
+	int "Max number of AEC channels"
 	default 2
 	help
-	  Sets the number AEC reference channels in the Google real-time
-	  communication audio processing.
+	  Sets the maximum number source/sink channels Google Real
+	  Time Communication Audio Processing will use for.  This is a
+	  computation and memory budget tunable.  Channel counts are
+	  retrieved at runtime, but channels higher than this number
+	  are ignored (on input) or cleared (output).
 
 config COMP_GOOGLE_RTC_AUDIO_PROCESSING_MEMORY_BUFFER_SIZE_BYTES
 	depends on COMP_GOOGLE_RTC_AUDIO_PROCESSING
```

Comment:
I think we want the memory size to be 351000 as we want to add more processing onboard the DSP.